src/components/Header/Tools.tsx

```diff
@@ -187,6 +188,12 @@ const Tools = () => {
         },
       ]
     : [
+        {
```

Comment:
This should also be enabled if Help panel is enabled. We should move it and show/hide it similarly as to how `enableGlobalLearningResourcesPage` is used.